components/developer/rust/rustc.p5m

```diff
@@ -11,7 +11,7 @@
 
 #
 # Copyright 2018 Aurelien Larcher
-# Copyright 2019 Michal Nowak
+# Copyright 2020 Michal Nowak
```

Comment:
Have you really meant this or wanted to add your 2020 copyright?

Reply:
Half intentional half missed mine.
@Mno-hime deserves a lot of the credit for this, as 1.40 is initially imported by him. I only figured out the problem with symlink cloning some time ago and wanted to try if I was right. As I had to copy his work with cp rather than git I wanted to keep at least some credit where credit is due.

Reply:
OK